console/console-init/ui/src/modules/iot-device/utils/constant.ts

```diff
@@ -97,6 +97,12 @@ const getLabelByValue = (typeValue: string) => {
   return filteredType.label as string;
 };
 
+enum DialogTypes {
```

Comment:
We should also add Update/Edit here and make this component generic across all modules.

Reply:
Okay, that sounds fair.